openwisp_monitoring/device/static/monitoring/css/monitoring.css

```diff
@@ -47,3 +47,25 @@ td.field-health_status,
 .health-ok{ color: green }
 .health-problem{ color: orange }
 .health-critical{ color: red }
+
+table {
```

Comment:
it's not a good idea to use `table` without referring it to a class or id, because it will be applied to any table, although this css is loaded only in a few pages so it won't affect the rest of the application, but do not get used to bad habits.

Reply:
I was going to change it as I noticed with it the device change list displays terribly :laughing:, expected after all it is also a `table`